README.md

```diff
@@ -1,15 +1,16 @@
 # Models
 
-[![Latest](https://img.shields.io/badge/docs-latest-blue.svg)](https://invenia.pages.invenia.ca/research/Models.jl/)
-[![Coverage](https://gitlab.invenia.ca/invenia/Models.jl/badges/master/coverage.svg)](https://gitlab.invenia.ca/invenia/Models.jl/commits/master)
+[![Build Status](https://travis-ci.com/invenia/Models.jl.svg?branch=master)](https://travis-ci.com/invenia/Models.jl)
+[![Build Status](https://ci.appveyor.com/api/projects/status/github/invenia/Models.jl?svg=true)](https://ci.appveyor.com/project/invenia/Models-jl)
+[![Codecov](https://codecov.io/gh/invenia/Models.jl/branch/master/graph/badge.svg)](https://codecov.io/gh/invenia/Models.jl)
+[![PkgEval](https://juliaci.github.io/NanosoldierReports/pkgeval_badges/N/Models.svg)](https://juliaci.github.io/NanosoldierReports/pkgeval_badges/report.html)
+[![code style blue](https://img.shields.io/badge/code%20style-blue-4495d1.svg)](https://github.com/invenia/BlueStyle)
```

Comment:
@oxinabox likes badges that don't change colour to be on a different line to those that do
```suggestion

[![code style blue](https://img.shields.io/badge/code%20style-blue-4495d1.svg)](https://github.com/invenia/BlueStyle)
```